src/journal/models.py

```diff
@@ -886,7 +896,7 @@ def __str__(self):
 @receiver(post_save, sender=Journal)
 def setup_default_section(sender, instance, created, **kwargs):
     if created:
-        submission_models.Section.objects.language('en').get_or_create(
+        submission_models.Section.objects.get_or_create(
```

Comment:
we should save the value of name for settings.LANGUAGE_CODE, otherwise creating a journal from a different language than the default will leave us without a default language entry